lib/PuppeteerSharp.Tests/Page/SetJavaScriptEnabledTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class SetJavaScriptEnabledTests : PuppeteerBaseTest
+    {
+        [Fact]
+        public async Task ShouldWork()
+        {
+            var page = await Browser.NewPageAsync();
```

Comment:
Can we use `using(var Page =...`?

Reply:
we don't really need to since `Browser.Dispose()` is called at the end